src/agents/common/state.py

```diff
@@ -28,9 +28,12 @@ class SubTask(BaseModel):
     description: str = Field(
         description="user query with original wording for the assigned agent"
     )
+    task_title: str = Field(
+        description="""Generate a title for 4 to 5 words, only use these:
```

Comment:
```suggestion
        description="""Generate a title of 4 to 5 words, only use these:
```
Just to double check: You try to say something like "Generate a title that consists out of 4 to 5 words", right?